python-threatexchange/tests/hashing/test_tlsh_hasher.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import base64
+import pathlib
+import tempfile
+import unittest
+
+from threatexchange.signal_type import tlsh_pdf
+
+RANDOM_PDF_TLSH = (
+    "T145B2859FE708266211A3026277C7AEE5FF76402C636AD5BA2C2CC11C23A1F2957773D5"
+)
+
+
+class TLSHHasherModuleUnitTest(unittest.TestCase):
+    def test_tlsh_from_file(self):
+        tlsh_complete_data_hash = tlsh_pdf.TLSHSignal.hash_from_file(
+            "data/test_pdf_complete.pdf"
+        )
+        tlsh_half_data_hash = tlsh_pdf.TLSHSignal.hash_from_file(
```

Comment:
Meant to be a check for matching since that exact hash isn't in the temp_db so I'm making sure the whole diffxlen function and threshold check is working. (The half pdf is just the complete with only the first half of the pages so we should be getting a match if that scenario ever happens irl)